### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by benbenlaw** [tterrag1098/TreeGrowingSimulator2014#15]
Its showing particles like its working but nothing is happening

**Comment by tterrag1098** [comment]
Are you sure it's not working? It's random, just like bonemeal, it might take a while especially with a lot of nearby saplings.

**Comment by rafacost** [comment]
Yeah, actually I can't see any functionality other than the particles on 1.15. Spend a whole minecraft day spamming shift and no joy.
Also there is no way to config sensitivity on the config file.
